src/gensxg.c

```diff
@@ -402,7 +402,7 @@ static void dump_options(const Options* opt) {
   fprintf(stderr, " publicKey: %s\n", opt->public_key);
   fprintf(stderr, " privateKey: %s\n", opt->private_key);
   fprintf(stderr, " date: %ld\n", opt->date);
-  fprintf(stderr, " expire: %ld\n", opt->expire);
+  fprintf(stderr, " expire: %ld\n", opt->duration);
```

Comment:
isn't it `opt->date + opt->duration` (with appropriate casting) ?

Reply:
Done.